packages/magento-product/components/ProductPageBreadcrumb/ProductPageBreadcrumb.tsx

```diff
@@ -0,0 +1,34 @@
+import { usePrevPageRouter } from '@graphcommerce/framer-next-pages'
+import { Breadcrumbs, Link, Typography } from '@mui/material'
+import { productPageCategory } from '../ProductPageCategory/productPageCategory'
+import { ProductPageBreadcrumbFragment } from './ProductPageBreadcrumb.gql'
+
+export function ProductPageBreadcrumb(props: ProductPageBreadcrumbFragment) {
```

Comment:
The component should forward the BreadcrumbsProps to the Breadcrumbs component so it can be styled from the outside.